code/modules/mob/living/silicon/mommi/mommi.dm

```diff
@@ -317,4 +329,16 @@ They can only use one tool at a time, they can't choose modules, and they have 1
 	radio.interact(src)//Just use the radio's Topic() instead of bullshit special-snowflake code
 
 /mob/living/silicon/robot/mommi/process_killswitch()
-	return
\ No newline at end of file
+	return
+
+/mob/living/silicon/robot/mommi/New()
+	udder = new(50)
+	udder.my_atom = src
+	..()
+
+/mob/living/silicon/robot/mommi/Life()
+	if(timestopped)
```

Comment:
You probably won't need this if you were to do
`if(..())
   if(udder && prob(5))`